manifests/plugin/pulp/params.pp

```diff
@@ -1,12 +1,16 @@
 # Default parameters for the Pulp smart proxy plugin
 class foreman_proxy::plugin::pulp::params {
-  $enabled          = true
-  $listen_on        = 'https'
-  $version          = undef
-  $group            = undef
-  $pulpnode_enabled = false
-  $pulp_url         = "https://${::fqdn}/pulp"
-  $pulp_dir         = '/var/lib/pulp'
-  $pulp_content_dir = '/var/lib/pulp/content'
-  $mongodb_dir      = '/var/lib/mongodb'
+
+  include ::puppet::params
+
+  $enabled            = true
+  $listen_on          = 'https'
+  $version            = undef
+  $group              = undef
+  $pulpnode_enabled   = false
+  $pulp_url           = "https://${::fqdn}/pulp"
+  $pulp_dir           = '/var/lib/pulp'
+  $pulp_content_dir   = '/var/lib/pulp/content'
+  $puppet_content_dir = $::puppet::params::server_envs_dir
```

Comment:
This is against c3dd6f16. As this is only the default for that param, it should get set in that puppet-default block of foreman_proxy::params also and used here.